packages/target-decisioning-engine/src/contextProvider.js

```diff
@@ -164,16 +170,16 @@ function createTimingContext() {
  *
  * The TargetDecisioningEngine initialize method
  * @param { import("@adobe/target-tools/delivery-api-client/models/DeliveryRequest").DeliveryRequest } deliveryRequest
+ * @param { import("@adobe/target-tools/src/parsing").default } parseURLImpl
  * @return { import("../types/DecisioningContext").DecisioningContext }
  */
-export function createDecisioningContext(deliveryRequest) {
+export function createDecisioningContext(deliveryRequest, parseURLImpl) {
```

Comment:
you'll need to replace anywhere `parseURLImpl` is referenced with `parseDomainImpl` instead.  And in the `createDecisioningContext` set the default implementation to `parseDomainBasic`.  Here is an example:

```js
/**
 *
 * The TargetDecisioningEngine initialize method
 * @param { import("@adobe/target-tools/delivery-api-client/models/DeliveryRequest").DeliveryRequest } deliveryRequest
 * @param { import("../types/DecisioningContext").ParseDomainFunc } parseDomainImpl
 * @return { import("../types/DecisioningContext").DecisioningContext }
 */
export function createDecisioningContext(
  deliveryRequest,
  parseDomainImpl = parseDomainBasic
) {
  const { context = EMPTY_CONTEXT } = deliveryRequest;
  return {
    ...createTimingContext(),
    user: createBrowserContext(context),
    page: createPageContext(context.address, parseDomainImpl),
    referring: createReferringContext(context.address, parseDomainImpl),
    geo: createGeoContext(context.geo || {})
  };
}
```